xbmc/network/TCPServer.cpp

```diff
@@ -227,9 +227,13 @@ void CTCPServer::Announce(ANNOUNCEMENT::AnnouncementFlag flag,
                           const std::string& message,
                           const CVariant& data)
 {
+  size_t connectionsSize = m_connections.size();
```

Comment:
actually why do you need to store the vector size?
Probably it's better if you simple use:

```
if (m_connections.empty())
   return;
````
and leave `m_connections.size()` below